src/Component/Symbolizer/LineEditor/LineEditor.tsx

```diff
@@ -242,7 +246,7 @@ export class LineEditor extends React.Component<LineEditorProps> {
       <CompositionContext.Consumer>
         {(composition: Compositions) => (
           <div className="gs-line-symbolizer-editor" >
-            <Collapse bordered={false} defaultActiveKey={['1']} onChange={(key: string) => (null)}>
+            <Collapse bordered={false} defaultActiveKey={['1']} onChange={() => (null)}>
```

Comment:
Why is this `onChange` even needed? Did you try removing it?

Reply:
You're right, removed it in 1a88ee7